.eslintrc.yml

```diff
@@ -24,3 +24,5 @@ overrides:
 
   - files: [cordova-js-src/**/*.js]
     extends: '@cordova/eslint-config/browser'
+    rules:
+      no-var: 0
```

Comment:
This rule should be applied directly to `@cordova/eslint-config/browser` but in the mean time applied here. Will hope to reoslve this in a later patch release of  `@cordova/eslint-config`.